DEVELOPMENT_GUIDE.md

```diff
@@ -0,0 +1,221 @@
+
+### Make commands
+
+There are `make` commands that alias some of this functionality:
+
+-   `lint` -- Lints the spec and code
+-   `publish` -- Outputs the specification as a **single file** into the `build/` directory
+-   `serve` -- Serves a preview of the specification in human-readable format
+
+### Testing
+
+Each API and team is unique. We encourage you to use a `test/` folder in the root of the project, and use whatever testing frameworks or apps your team feels comfortable with. It is important that the URL your test points to be configurable. We have included some stubs in the Makefile for running tests.
+
+### VS Code Plugins
+
+-   [openapi-lint](https://marketplace.visualstudio.com/items?itemName=mermade.openapi-lint) resolves links and validates entire spec with the 'OpenAPI Resolve and Validate' command
+-   [OpenAPI (Swagger) Editor](https://marketplace.visualstudio.com/items?itemName=42Crunch.vscode-openapi) provides sidebar navigation
+
+### Emacs Plugins
+
+-   [**openapi-yaml-mode**](https://github.com/esc-emacs/openapi-yaml-mode) provides syntax highlighting, completion, and path help
+
+### Speccy
+
+> [Speccy](http://speccy.io/) _A handy toolkit for OpenAPI, with a linter to enforce quality rules, documentation rendering, and resolution._
+
+Speccy does the lifting for the following npm scripts:
+
+-   `test` -- Lints the definition
+-   `publish` -- Outputs the specification as a **single file** into the `build/` directory
+-   `serve` -- Serves a preview of the specification in human-readable format
+
+(Workflow detailed in a [post](https://developerjack.com/blog/2018/maintaining-large-design-first-api-specs/) on the _developerjack_ blog.)
+
+:bulb: The `publish` command is useful when uploading to Apigee which requires the spec as a single file.
+
+### Caveats
+
+#### Swagger UI
+
+Swagger UI unfortunately doesn't correctly render `$ref`s in examples, so use `speccy serve` instead.
+
+#### Apigee Portal
+
+The Apigee portal will not automatically pull examples from schemas, you must specify them manually.
+
+### Platform setup
+
+As currently defined in your `proxies` folder, your proxies do pretty much nothing.
+Telling Apigee how to connect to your backend requires a _Target Server_, which you should call named `validated-relationships-service-api-target`.
+Our _Target Servers_ defined in the [api-management-infrastructure](https://github.com/NHSDigital/api-management-infrastructure) repository.
+
+:bulb: For Sandbox-running environments (`test`) these need to be present for successful deployment but can be set to empty/dummy values.
+
+### Detailed folder walk through
+
+To get started developing your API use this template repo alongside guidance provided by the [API Producer Zone confluence](https://nhsd-confluence.digital.nhs.uk/display/APM/Deliver+your+API)
```

Comment:
Reword or remove given it pertains to the template and external contributors won't have access to the API producer docs in confluence (may well be fine to link to it, but perhaps a note to call out that it's internal NHSE guidance documentation)

Reply:
At the top of the document I've added a warning to state that the development guide contains links to NHS internal documentation